### PR TITLE
Move to PSR4 structure / Add docker compose

### PR DESCRIPTION
**Pull request by crosello**
@rcambien ;)

- Move to PSR4 structure
- Add docker compose

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -1,6 +1,11 @@
...
-<phpunit bootstrap="tests/bootstrap.php" colors="true">
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.1/phpunit.xsd"
+         backupGlobals="false"
+         bootstrap="vendor/autoload.php"
+         colors="true"
+>
     <php>
         <!-- AWS SQS -->
         <var name="AWS_ACCESS"        value="" />
```
Comment (rcambien) [inline phpunit.xml.dist]: Can you remove the unused variables ?

File: tests/Riverline/WorkerBundle/Provider/AwsSQSTest.php [hunk 2]
```diff
@@ -1,115 +0,0 @@
-<?php
```
Comment (rcambien) [inline tests/Riverline/WorkerBundle/Provider/AwsSQSTest.php:1]: Can you add in the AwsSQS class something to inform the user that the class is deprecated and no more tested ?

File: tests/Provider/AwsSQSv2Test.php [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 namespace Riverline\WorkerBundle\Provider;
 
-class AwsSQSv2Test extends \PHPUnit_Framework_TestCase
+abstract class AwsSQSv2Test extends \PHPUnit_Framework_TestCase
```
Comment (rcambien) [inline tests/Provider/AwsSQSv2Test.php]: Abstract ?

**Review by rcambien — CHANGES_REQUESTED** [review]

File: tests/Riverline/WorkerBundle/Provider/AwsSQSTest.php (hunk 2 above)
Comment (crosello) [reply tests/Riverline/WorkerBundle/Provider/AwsSQSTest.php:1]: Done

File: tests/Provider/AwsSQSv2Test.php (hunk 3 above)
Comment (crosello) [reply tests/Provider/AwsSQSv2Test.php]: Oops ;)

File: phpunit.xml.dist (hunk 1 above)
Comment (crosello) [reply phpunit.xml.dist]: Done

**Review by rcambien — APPROVED** [review]

**Pull request merged**